apps/__init__.py

```diff
@@ -1,3 +1,7 @@
+'''
```

Comment:
Pep257 recommends using double quotes for doc strings in python. Is there are reason for using single quotes?